bin/build_openmp.sh

```diff
@@ -85,7 +85,7 @@ else
   COMMON_CMAKE_OPTS="$COMMON_CMAKE_OPTS \
 -DLLVM_MAIN_INCLUDE_DIR=$AOMP_REPOS/$AOMP_PROJECT_REPO_NAME/llvm/include \
 -DLIBOMPTARGET_LLVM_INCLUDE_DIRS=$AOMP_REPOS/$AOMP_PROJECT_REPO_NAME/llvm/include \
--DLIBOMP_USE_HWLOC=ON -DLIBOMP_HWLOC_INSTALL_DIR=$AOMP_SUPP/hwloc"
+-DLIBOMP_USE_HWLOC=ON -DLIBOMP_HWLOC_INSTALL_DIR=$AOMP_SUPP/hwloc -DENABLE_DEVEL_PACKAGE=ON -DENABLE_RUN_PACKAGE=ON"
```

Comment:
-DENABLE_DEVEL_PACKAGE=ON -DENABLE_RUN_PACKAGE=ON, these options need to be moved up to only apply when AOMP_STANDALONE_BUILD == 0 .

Reply:
Done